docs/swagger.yaml

```diff
@@ -0,0 +1,381 @@
+      parameters:
+      - name: "user_id"
+        in: "path"
+        description: "The id of the user who owns the task."
+        required: true
+        type: "string"
+      - name: "task_id"
+        in: "path"
+        description: "The task id that will be updated."
+        required: true
+        type: "integer"
+      - in: "body"
+        name: "body"
+        description: "Updated user object"
+        required: true
+        schema:
+          $ref: "#/definitions/User"
+      responses:
+        "200":
+          description: "Successful operation"
+        "404":
+          description: "User not found"
+        "500":
+          description: "Internal server error"
+    patch:
+      tags:
+      - "users/{user_id}/tasks/{task_id}"
+      summary: "Update a user's task with PATCH method"
+      description: "This can only be done by the logged in user."
+      operationId: "patchTask"
+      produces:
+      - "application/json"
+      parameters:
+      - name: "user_id"
+        in: "path"
+        description: "The id of the user who owns the task."
+        required: true
+        type: "string"
+      - name: "task_id"
+        in: "path"
+        description: "The task id that will be updated."
+        required: true
+        type: "integer"
+      - in: "body"
+        name: "body"
+        description: "Updated user object"
+        required: true
+        schema:
+          $ref: "#/definitions/User"
+      responses:
+        "200":
+          description: "Successful operation"
+        "404":
+          description: "User not found"
+        "500":
+          description: "Internal server error"
+    delete:
+      tags:
+      - "users/{user_id}/tasks/{task_id}"
+      summary: "Delete a user"
+      description: "This can only be done by the logged in user."
+      operationId: "deleteTask"
+      produces:
+      - "application/json"
+      parameters:
+      - name: "user_id"
+        in: "path"
+        description: "The user id of the owner of the task that will be deleted"
+        required: true
+        type: "string"
+      - name: "task_id"
+        in: "path"
+        description: "The id of the task that will be deleted"
+        required: true
+        type: "integer"
+      responses:
+        "200":
+          description: "Successful operation"
+        "400":
+          description: "User not found"
+        "500":
+          description: "Internal server error"
+
+definitions:
+  Task:
+    type: "object"
+    properties:
+      userId:
+        type: "integer"
+        format: "int64"
+      user:
+        type: "string"        
+      taskId:
+        type: "integer"
+        format: "int64"
+      description:
+        type: "string"        
+      quantity:
+        type: "integer"
+        format: "int32"
+      createdDate:
+        type: "string"
+        format: "date-time"
+      dueDate:
+        type: "string"
+        format: "date-time"
+      completed:
+        type: "integer"
```

Comment:
I think this actually needs to be a string? When I tested this with "completed": 0, I got a failure - it only worked with "completed": "0"